apps/app/src/preload/api.ts

```diff
@@ -29,4 +30,5 @@ export const api = {
   appHelp,
   appUpdater,
   jsonStore,
+  usbAccessPreload,
```

Comment:
Please keep consistent naming for the `window.api` interface

```suggestion
  usbAccess: usbAccessPreload,
```

Reply:
https://github.com/mudita/mudita-center/pull/2578/commits/239434845333cdf009957f2a0d7e36453bd5e4f9